pkg/operatorsource/handler.go

```diff
@@ -123,3 +137,18 @@ func (h *operatorsourcehandler) transition(ctx context.Context, logger *log.Entr
 
 	return reconciliationErr
 }
+
+// updateMetrics takes the name of the operator source which fails to reconcile, and
+// increments the error count of the respective operator metric.
+func updateMetrics(opsrc string) {
+	switch opsrcName := opsrc; opsrcName {
+	case REDHAT_OPERATOR:
+		customMetrics.CounterForRedHatOpsrcError(opsrcName).Inc()
```

Comment:
Is the implementation of `Inc()` handled by prometheus? I am just curious because usually the first example of distributed programming errors is one of incrementing counters lol

Reply:
True!. Fortunately, Prometheus is handling that here.